assets/prototype/domain/shared/predicates/tickets/isPending.ts

```diff
@@ -1,17 +1,18 @@
 /**
  * External dependencies
  */
-import { differenceInMinutes } from 'date-fns';
+import { differenceInMinutes, parseISO } from 'date-fns';
 import { now } from './filters';
+import { Ticket } from '../../../eventEditor/data/types';
 
 /**
  * @function
  * @param {Object} ticket object
  * @return {boolean} 	true if ticket is not yet available for purchase,
  * 						but will be at some date in the future
  */
-const isPending = ({ startDate }) => {
-	return differenceInMinutes(startDate, now) > 0;
+const isPending = ({ startDate }: Ticket) => {
+	return differenceInMinutes(parseISO(startDate), now) > 0;
```

Comment:
needs typing for return value

Reply:
Added: https://github.com/eventespresso/event-espresso-core/pull/2150/commits/b8bebcf79946133a296c6ad4cccd52bddb17ea85